mllib/pom.xml

```diff
@@ -109,7 +109,7 @@
     <dependency>
       <groupId>org.jpmml</groupId>
       <artifactId>pmml-model</artifactId>
-      <version>1.1.15</version>
+      <version>1.2.7</version>
```

Comment:
The `jpmml-model` is the top-level (ie. project) artifact. It provides "cover" for a list of modules, such as `pmml-model`, `pmml-schema`, `pmml-agent` etc.

If you want to include PMML class model then a single `pmml-model` dependency will suffice. If you declared `jpmml-model` dependency, then you would get a bunch of unnecessary modules included in your build.